### PR TITLE
added role object and its functions

### PR DESCRIPTION
**Pull request by ryytz**
Role object for anything makes it easier to use

### CODE REVIEW

**Comment by ryytz** [comment]
YOU FOOL! THIS IS NOT JAVA! FIX YOUR FUNCTION NAMES OR DIE BRAINLESS IDIOT!

File: GoblinSabotageSquad/role.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+from elf_kingdom import *
+
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py:1]: why?

File: GoblinSabotageSquad/role.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+class Role:
+    """
+
+    """
+    global role
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py]: this should not be global, this should be stored in self

File: GoblinSabotageSquad/role.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+    def setRole(self, role):
+        self.role =  role
+
+    def getRole(self):
+        return self.role
+
+    def compare(self, roleB):
+        if (roleB == self):
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py]: `return roleb == self.role`

File: GoblinSabotageSquad/role.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    def isAttacker(self):
+        global role
+        if(role == "Attacker"):
+            return True
+        return False
+
+    def isDefender(self):
+        global role
+        if(role == "Defender"):
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py]: `return role == "Defender"`

File: GoblinSabotageSquad/role.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            return True
+        return False
+
+    def isReserve(self):
+        global role
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py]: you know

File: GoblinSabotageSquad/role.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        if(role == "Reserve"):
+            return True
+        return False
+
+    def isPortal(self):
+        global role
+        if(role == "Portal"):
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py]: you know

File: GoblinSabotageSquad/role.py [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+            return True
+        return False
+
+    def isCannibal(self):
+        global role
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py]: y k

File: GoblinSabotageSquad/role.py [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+        if(role == "Cannibal"):
+            return True
+        return False
+
+    def isDestroyer(self):
+        global role
```
Comment (ScarletMetal) [inline GoblinSabotageSquad/role.py]: y k

**Review by ScarletMetal — CHANGES_REQUESTED** [review]

**Review by ScarletMetal — APPROVED** [review]

**Pull request merged**